front/plugins/_publisher_apprise/apprise.py

```diff
@@ -78,7 +78,7 @@ def main():
 
 #-------------------------------------------------------------------------------
 def check_config():
-        if get_setting_value('APPRISE_URL') == '' or get_setting_value('APPRISE_HOST') == '':            
+        if get_setting_value('APPRISE_HOST') == '' or get_setting_value('APPRISE_URL') == '':            
```

Comment:
Unsure why this has to change?

Reply:
I doesn't, I was originally validating the new setting there but decided against it and I guess this was left-over, I can revert this line

Reply:
Thanks! :)